src/pymmcore_widgets/_mda/_stack_viewer.py

```diff
@@ -0,0 +1,284 @@
+import copy
+
+import numpy as np
+from pymmcore_plus import CMMCorePlus
+from qtpy import QtCore, QtGui, QtWidgets
+from useq import MDASequence
+
+from pymmcore_widgets._mda._datastore import QLocalDataStore
+from pymmcore_widgets._mda._util._channel_row import ChannelRow
```

Comment:
if vispy wasn't installed, this import of ChannelRow would cause an `ImportError` without the useful message below.  that needs to be resolved

Reply:
Added the try/except to the channel row. We could think about not using vispy there at all, but just QColors or something, then translate to vispy colormaps only in the StackViewer.

Reply:
agreed.  could also consider using `cmap` (https://cmap-docs.readthedocs.io/en/latest/) ... which is a minimal-dependency library I created to be able to use colormaps without assuming a specific platform (works for matplotlib, vispy, napari, etc...)